commit f9c6b4c7459c
Author: Lee Salzman <lsalzman@mozilla.com>
Date:   Fri Sep 29 23:59:40 2017 -0400

    Bug 1404323 - Limit WindowSurfaceX11Image::Commit to clip bounds. r=gankro, a=ritu
    
    MozReview-Commit-ID: 55GbZ2EHdbQ
    
    --HG--
    extra : source : 19c2892e7a3ad24b6d235948ef514b20e216b57d
---
 widget/gtk/WindowSurfaceX11Image.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git widget/gtk/WindowSurfaceX11Image.cpp widget/gtk/WindowSurfaceX11Image.cpp
index 5de0288043d6..59db17b642f2 100644
--- widget/gtk/WindowSurfaceX11Image.cpp
+++ widget/gtk/WindowSurfaceX11Image.cpp
@@ -93,7 +93,7 @@ WindowSurfaceX11Image::Commit(const LayoutDeviceIntRegion& aInvalidRegion)
   }
 
   gfx::IntRect bounds = aInvalidRegion.GetBounds().ToUnknownRect();
-  gfx::Rect rect(0, 0, bounds.XMost(), bounds.YMost());
+  gfx::Rect rect(bounds);
   if (rect.IsEmpty()) {
     return;
   }
